docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -1,12 +1,43 @@
-﻿using Samples.Dynamic;
+﻿using System;
+using System.Reflection;
+using Samples.Dynamic;
 
 namespace Microsoft.ML.Samples
 {
-    internal static class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
             CalculateFeatureContribution.Example();
+
+            if (args.Length > 0 && args[0] == "-runall")
```

Comment:
>if (args.Length > 0 && args[0] == "-runall") [](start = 12, length = 44)

we want this run all samples without any arguments, so that devs can just hit F5 while creating samples w/o having to mock around with debug arguments.